### ISSUE TITLE
Suggestion: import metadatastore.doc instead of bundling it

### ISSUE THREAD

**Issue by danielballan** [cowanml/synch_service_utils#1]
I took a very quick look today. Instead of copy/pasting and bundling code from metadatastore to get doc.py, I suggest introducing a dependency on metadatastore.

Forgive me if you had a reason for doing it this way that I just didn't pick up on.

**Comment by cowanml** [comment]
synch_service_utils was an attempt to extract all the common code into
a library all services could depend on.  Was exactly trying to avoid
copy/pasting+bundling and interpackage dependencies :)

**Comment by danielballan** [comment]
Fair enough. Note that other uses of doc.py include clients that don't need tornado so this will probably be pulled into a stand-alone package, not bundled with server code.

Also, just FYI, when moving code is it useful to retain the version history instead of copy/pasting the current state of the file.
